include/s3select.h

```diff
@@ -11,7 +11,7 @@
 #include <list>
 #include "s3select_oper.h"
 #include "s3select_functions.h"
-#include "s3select_csv_parser.h"
+#include "csv.h"
```

Comment:
should define `CSV_IO_NO_THREAD` to avoid using thread

Reply:
done